packages/SystemUI/src/com/android/systemui/statusbar/phone/NavigationBarView.java

```diff
@@ -712,7 +712,7 @@ public void setLayoutTransitionsEnabled(boolean enabled) {
     }
 
     public void setWakeAndUnlocking(boolean wakeAndUnlocking) {
-        setUseFadingAnimations(wakeAndUnlocking);
+        setUseFadingAnimations(!wakeAndUnlocking);
```

Comment:
Can you explain this part?

Reply:
Im not sure why that is added. It was part of resolving the conflict I believe